packages/types/src/interview.ts

```diff
@@ -128,8 +131,8 @@ export type FindPagedInterviewsProps = {
     InfiniteData<
       Paginated<{
         interview: IInterview.Self;
-        call: ICall.Self;
-        members: ICall.PopuldatedMember[];
+        session: ISession.Self;
+        members: ISession.PopuldatedMember[];
       }>,
       unknown
```

Comment:
We can remove this type `FindPagedInterviewProps` becuase of two reasons. 

- This package is low level types, it has nothing to do with `Props` (of react)
- It makes use of a package that can not be used on the server which is `@tanstack/react-query`. 

Please remove this type as well as the `@tanstack/react-query` package.